components/twitch_developer_app/app/common/common.ts

```diff
@@ -35,7 +36,7 @@ export default {
     },
   },
   methods: {
-    async manageWebHooks(type: string, conditions: any) {
+    async manageWebHooks(type: string, conditions: Condition[]) {
```

Comment:
Is this the correct way to do the type?

Reply:
I think this seems right. Let me check it once again if it is ready